### ISSUE TITLE
Incorrect Output Window Behavior

### ISSUE THREAD

**Issue by BrainSlugs83** [Microsoft/vsminecraft#14]
Instead of logging to the output window (like Eclipse or other IDEs do with other Minecraft mods -- or even like you would expect Visual Studio to do) instead of that, a new console window comes up and the output is displayed there.

Instead, that console stream should be redirected to the output window.  (Normal applications in Visual Studio log to the output window instead of creating new console windows -- Note: if you don't want to lose the other data that's being logged to the "Debug" source in the output window, you can create a new source for the output window).

First screenshot shows the Console Window being up on the display and the type of data that's both in the Console window and the "Debug" source of the output window.

![Console Window & Output Window](https://i.imgur.com/Ezw3Eqx.png)

Second screenshot shows an example of other sources for the Output window. -- Ideally, you could just create a "Minecraft" source and redirect stdout from "java.exe".

![Output Window Sources](https://i.imgur.com/78yt2vq.png)

When you implement this, please set the Output window's source to "Minecraft" by default when the debugger is attached.

**Comment by sharwell** [comment]
@BrainSlugs83 This really is a great suggestion, but the particular feature you are requesting is fully implemented by a different project. Do you mind creating a new issue here for it:

https://github.com/tunnelvisionlabs/JavaForVS/issues

**Comment by mluparu** [comment]
Yep, agree @sharwell. Great suggestion. (Marked as external in this repo for now)

**Comment by BrainSlugs83** [comment]
Thanks guys, I'll log it there.

**Comment by BrainSlugs83** [comment]
It's been [re-posted](https://github.com/tunnelvisionlabs/JavaForVS/issues/26).

**Comment by BrainSlugs83** [comment]
Hmmm... guess @sharwell marked it as a "wontfix". -- Very strange given the comments here.  (Didn't realize it was the same person in both threads until just now.) -- Oh well, I guess sometimes people change their minds. :-/

**Comment by sharwell** [comment]
@BrainSlugs83 If I changed it once, I could change it again. Your arguments make sense in both issues. I invited you to join the gitter chat room for the **JavaForVS** repository.

**Comment by BrainSlugs83** [comment]
Thanks, @sharwell, I appreciate the open mindedness. :-) -- I've joined the chat room (I didn't know there were github chat rooms!).  I have plans for the evening but I'll keep it open in my browser and I will check on it periodically while I'm around. :-)
